modules/javafx.controls/src/main/java/javafx/scene/control/skin/TreeCellSkin.java

```diff
@@ -207,7 +207,7 @@ public final DoubleProperty indentProperty() {
         boolean disclosureVisible = disclosureNode != null && treeItem != null && ! treeItem.isLeaf();
 
         final double defaultDisclosureWidth = maxDisclosureWidthMap.containsKey(tree) ?
-                maxDisclosureWidthMap.get(tree) : 18;   // JDK-8119169: default width of default disclosure node
+                maxDisclosureWidthMap.get(tree) : 20;   // JDK-8119169: default width of default disclosure node
```

Comment:
This is probably not a good solution: I think the root cause is not in that the default value is incorrect, but in that two different values are used in the same view (i.e. that it first uses the default value, then the value computed from the disclosure node, if I read the code correctly).

Could you please investigate?

I've linked the JBS ticket mentioned in the comment, take a look at the discussion there https://bugs.openjdk.org/browse/JDK-8119169 , especially the comment about variable width disclosure node.